02_configure_host.sh

```diff
@@ -127,7 +127,8 @@ reg_state=$(sudo "$CONTAINER_RUNTIME" inspect registry --format  "{{.State.Statu
 
 # ubuntu_install_requirements.sh script restarts docker daemon which causes local registry container to be in exited state.
 if [[ "$reg_state" == "exited" ]]; then
-  sudo "${CONTAINER_RUNTIME}" start registry  
+  sudo "${CONTAINER_RUNTIME}" start registry
+  sleep 5  
```

Comment:
Should not it be outside of the condition as you did earlier? Because we might have the similar issue not only when we restart the registry but also when we start it initially?

Reply:
This issue occurs only when we restart the registry. In new environment/VM it always create new registry and in this case will not have this issue. For example if you see all the CI run, we never face this issue. Also I have tried with new VM and I didn't face any error. So in CI we will not see any delays.